packages/server/src/env.ts

```diff
@@ -0,0 +1,8 @@
+export const PORT = 3001;
```

Comment:
I think this is usually in .env file but yeah we can have this file here anyway maybe we can call it constants instead of env?

Reply:
ok rephrasing this, I thikn this all thing should go in .env you can have different env like env.production env.local etc so it would be better and no if involved